Makefile

```diff
@@ -132,6 +116,14 @@ proto: ratelimit/gubernator/proto/google ratelimit/gubernator/gubernator.proto $
 	@cp -f $(PROTOC_GEN_GO) $(BIN_DIR)/protoc-gen-go
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin scripts/generate_proto.sh
 
+.PHONY: container-test
+container-test: build
+	@cp ./observatorium-api ./observatorium_api_for_docker
+	@docker build \
+		-f Dockerfile.e2e-test \
+		-t $(DOCKER_REPO):latest .
+	@rm ./observatorium_api_for_docker
```

Comment:
Why do we need this extra copy and rm here?

Reply:
We don't, you're right, it's completely meaningless here, removing.